stac_mjx/utils.py

```diff
@@ -187,11 +187,19 @@ def _load_params(param_path):
             print(exc)
     return params
 
+def load_stac_tranform(save_path): 
+    _, file_extension = os.path.splitext(save_path)
+    if file_extension == ".p":
+        with open(save_path, "rb") as file:
+            fit_data = pickle.load(file)
+    elif file_extension == ".h5":
```

Comment:
Eh, why did we need load from h5?

Reply:
Because that is how I have the data. Pickles in general are not a great way to store data. They require the exact state of software and are insecure. You can run arbitrary code in pickles and it is recommended to store data in other formats.